.github/workflows/stale-actions.yml

```diff
@@ -0,0 +1,24 @@
+name: "Mark or close stale issues and PRs"
+on:
+  schedule:
+  - cron: "00 10 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v8
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 30
+        days-before-close: 30
```

Comment:
I think `days-before-close` can be shorter, such as `7`.

Reply:
I agree with you, I've fixed it.